RELEASE_NOTES.md

```diff
@@ -1,3 +1,49 @@
+# v2022.03.0
+
+## Changes Impacting Builds
+This release of Umpire contains new build requirements including:
+
+- C++14 is now required to build Umpire
+- `cmake` version 3.14 or later is required
+
+## Changes Impacting C/Fortran
+- The `cmake` object library for `C/FORTRAN` interface has been reorganized.
+  (**NOTE**: *This is a breaking change since the include paths are now different.*)
+
+## New Interfaces
+- Added a `getDeviceAllocator` function that allows users to get a `DeviceAllocator` object from the kernel without explicitly passing the allocator to the kernel first.
+- Added a `reset` function to the `DeviceAllocator` so that old data can be rewritten.
+- Expose `PREFETCH` operations registered with the `MemoryOperationRegistry` with a new `ResourceManager::prefetch` method.
+
+## Removed Interfaces
```

Comment:
I think there should be a blank line after all these headings.

Reply:
Done